### PR TITLE
Add minor improvements

### PR DESCRIPTION
**Pull request by pranavan15**
## Purpose
This PR contains the below modifications,
- Add missing test case for travel agency service
- Remove unnecessary Before and After suites in test files
- Replace null with () in JSON parsing validations and map accesses
- Include all the tests in Travis
- Update next release version in Travis

This is compatible with Ballerina version 0.981.2-SNAPSHOT

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -178,7 +178,7 @@ fork {
     int emiratesPrice;
 
     // Get the response and price for airline 'Qatar Airways'
-    if (airlineResponses["qatarWorker"] != null) {
+    if (airlineResponses["qatarWorker"] != ()) {
```
Comment (riyafa) [inline README.md]: Shall we use nil lifting or match statements instead of checking using if statements in all places?

Comment (pranavan15) [reply README.md]: We can actually eliminate this check in all the `fork-join` operations those join all the results. However it is needed for the conditional join operation to check which worker responses first.

And in the json parsing validations, if we are to match each element separately we will end up duplicating the code segment that sends a response with `bad request` message. Hence, it's checked using if conditions only after parsing all the elements.

File: README.md [hunk 2]
```diff
@@ -178,33 +178,27 @@ fork {
...
-        jsonFlightResponseQatar = check resQatar.getJsonPayload();
-        match jsonFlightResponseQatar.Price {
-            int intVal => qatarPrice = intVal;
-            any otherVals => qatarPrice = -1;
-        }
+    var resQatar = check <http:Response>(airlineResponses["qatarWorker"]);
+    jsonFlightResponseQatar = check resQatar.getJsonPayload();
+    match jsonFlightResponseQatar.Price {
+        int intVal => qatarPrice = intVal;
+        any otherVals => qatarPrice = -1;
```
Comment (riyafa) [inline README.md]: This is the use case of but operation? Shall we try that?

Comment (pranavan15) [reply README.md]: Ack; Addressed as suggested.

**Pull request merged**